app/cafe.py

```diff
@@ -0,0 +1,26 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("NotVaccinatedError")
+
+        today_date = datetime.date.today()
+        if today_date > visitor["vaccine"]["expiration_date"]:
+            raise OutdatedVaccineError("OutdatedVaccineError")
+
+        if not visitor.get("wearing_a_mask"):
+            raise NotWearingMaskError("NotWearingMaskError")
```

Comment:
provide more detailed exception description like "Customer should be vaccinated"